### PR TITLE
[Project Submission] Heavenly Jungle NFT

### PR DESCRIPTION
**Pull request by JFF-Danno**

### CODE REVIEW

File: projects/Heavenly Jungle NFT [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+## Target Audience
+Art Communities, Individual Artists
+
+## Rough estimated user base (if applicable)
+<!--How many users do you have right now?-->
+
+## Github repo
+<!--Attach a link to your GitHub repo - open source is required - please make sure your repo has a license file and is licensed using MIT open source license! -->
+https://github.com/Mol-LeArt/Heavenly-Jungle
```
Comment (autonome) [inline projects/Heavenly Jungle NFT:26]: Source code repo is missing license file

**Review by autonome — CHANGES_REQUESTED** [review]
Your project source code must be licensed MIT in order to qualify for the grant program.

**Comment by JFF-Danno** [comment]
Thanks Dietrich,

I've add a license file.
https://github.com/Mol-LeArt/Heavenly-Jungle/blob/main/LICENSE

**Review by autonome — CHANGES_REQUESTED** [review]
Can you change the filename to have .md extension?

**Comment by JFF-Danno** [comment]
No problem I have done.

File: projects/Heavenly Jungle NFT [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+## Website
+<!--Link your website if available-->
+
+## Docs
+<!--Including a link to your project docs!-->
+
+## Team Info
+<!-- Introduce your amazing team - how many team members are working on this project and who are they?-->
+Daniel - front end dev
```
Comment (autonome) [inline projects/Heavenly Jungle NFT:36]: Please link the Github accounts of team members

**Review by autonome — CHANGES_REQUESTED** [review]

File: projects/Heavenly Jungle NFT (hunk 2 above)
Comment (JFF-Danno) [reply projects/Heavenly Jungle NFT:36]: Daniel - https://github.com/JFF-Danno
Stamford - https://github.com/audsssy

File: projects/Heavenly Jungle NFT [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+DAO NFT Platform allowing fractional ownership. Includes minting, auctions, royalties, licensing and the option to rage quit.
+
+## Use of IPFS, Filecoin and Libp2p
+Currently using Pinata's IPFS pinning service. Looking for guidance on this matter.
+
+## Project Status
+Early Development
+
+## Previews
+<!--Add some screenshots to give a preview of your product-->
```
Comment (autonome) [inline projects/Heavenly Jungle NFT:16]: Please add screenshot

**Review by autonome — CHANGES_REQUESTED** [review]

File: projects/Heavenly Jungle NFT (hunk 3 above)
Comment (JFF-Danno) [reply projects/Heavenly Jungle NFT:16]: NFT View
About Commons
Administate the Commons
Commons Gallery

**Pull request merged**